ocf_data_sampler/torch_datasets/process_and_combine.py

```diff
@@ -37,18 +39,32 @@ def process_and_combine_datasets(
             # Standardise
             provider = config.input_data.nwp[nwp_key].provider
             da_nwp = (da_nwp - NWP_MEANS[provider]) / NWP_STDS[provider]
+
             # Convert to NumpyBatch
             nwp_numpy_modalities[nwp_key] = convert_nwp_to_numpy_batch(da_nwp)
 
         # Combine the NWPs into NumpyBatch
         numpy_modalities.append({NWPBatchKey.nwp: nwp_numpy_modalities})
 
+
     if "sat" in dataset_dict:
-        # Satellite is already in the range [0-1] so no need to standardise
-        da_sat = dataset_dict["sat"]
 
-        # Convert to NumpyBatch
-        numpy_modalities.append(convert_satellite_to_numpy_batch(da_sat))
+        sat_numpy_modalities = dict()
+
+        for sat_key, da_sat in dataset_dict["sat"].items():
```

Comment:
did this work? Im surprised as I thought we only have one satellite unlike NWP where we have mulitple. This is basically because we have only ever used one satellite in one area, compared to using multie NWP for a model. 

Note in when loading datasets, satellite seems to just be one - https://github.com/openclimatefix/ocf-data-sampler/blob/main/ocf_data_sampler/load/load_dataset.py#L42

Reply:
Yeh that is true, the satellite datasets aren't nested like the NWPs are since we only use one satellite product